server/authz/policy.rego

```diff
@@ -657,11 +657,11 @@ allow {
   action == write
 }
 
-# Team admins, maintainers, observers, and observer_plus can read any software installer in their teams.
+# Team admins and maintainers can read any software installer in their teams.
 allow {
   not is_null(object.team_id)
   object.type == "software_installer"
-  team_role(subject, object.team_id) == [admin, maintainer, observer, observer_plus][_]
+  team_role(subject, object.team_id) == [admin, maintainer][_]
```

Comment:
I think the problem (and the confusing part) is that they can read installers, but they can't download them, I believe this will restrict both actions.

Reply:
I was still able to read (list and access the details page) fwiw; I think those actions are permissioned using the software titles object, which allows the access

Reply:
good point, thanks! how do you see that playing out with the upcoming changes in https://github.com/fleetdm/fleet/issues/19561?